package.json

```diff
@@ -33,5 +33,10 @@
   },
   "engines": {
     "node": ">=14"
+  },
+  "dependencies": {
+    "@vue/compiler-sfc": "3.2.45",
```

Comment:
Just out of curiosity: can you tell me what package required that? Needing a Vue.js compiler in a React project seams kinda sus.